internal/rest/resources/tokens.go

```diff
@@ -58,6 +59,14 @@ func tokensPost(state *state.State, r *http.Request) response.Response {
 		joinAddresses = append(joinAddresses, addr)
 	}
 
+	if len(joinAddresses) == 0 {
```

Comment:
Thanks for this fix.

Do we need to have this "workaround" though? Can we not wait for the  trust store to become ready?

Reply:
I think this is still useful, as we might not be able to reach the other nodes of the truststore for whatever reason. This at least guarantees that since we got the token request, we can answer it and validate it.